typed-racket-lib/typed-racket/rep/type-rep.rkt

```diff
@@ -579,6 +579,17 @@
   [#:frees (λ (f) (make-invariant (f value)))]
   [#:key 'continuation-mark-key])
 
+;; Newtype
+;; comes from define-new-subtype
+;; nm: a symbol representing the name of the Newtype
+;; id: a symbol created with gensym
+;; ty: a type for the representation
+(def-type Newtype ([nm symbol?] [id symbol?] [ty Type/c])
```

Comment:
I've considered that point, too, and I'm ambivalent about it. I feel like explicitly including the word "new" in `define-new-subtype` adds a little bit of semantic information wrt intent, but on the other hand, it does feel redundant.